jcl/src/openj9.sfj/share/classes/module-info.java

```diff
@@ -1,6 +1,6 @@
 /*[INCLUDE-IF Sidecar19-SE]*/
 /*******************************************************************************
- * Copyright (c) 2016, 2017 IBM Corp. and others
+ * Copyright (c) 2017, 2017 IBM Corp. and others
```

Comment:
This looks incorrect

Reply:
This one isn't actually a rename like Github thinks. I added the `module-info.java` file for this module to OpenJ9 so our old build system can process it and remove the line that exports to the old `com.ibm.management` name. So this file is completely new to OpenJ9.